dnf-testing.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import argparse
+import os
+import re
+import subprocess
+import sys
+
+DESTRUCTIVE_TAGS = set([
+    "@destructive",
+    "@no_installroot",
+])
+PROGPATH = os.path.abspath(os.path.dirname(sys.argv[0]))
+
+
+class MultilineHelpFormatter(argparse.HelpFormatter):
+    '''
+    Help formatter that allows multiline help texts.
+    '''
+    def _split_lines(self, text, width):
+        if '\n' in text:
+            return text.splitlines()
+        return super(MultilineHelpFormatter, self)._split_lines(text, width)
+
+def command_line_parser():
+    '''
+    Initialize and return command line arguments parser.
+    '''
+    parser = argparse.ArgumentParser(
+        description="Functional tests for DNF",
+        formatter_class=MultilineHelpFormatter)
+    parser.add_argument(
+        "--featuresdir", default="features",
+        help="Directory with *.feature files")
```

Comment:
So... how about calling this option `"-s", "--suite"`, as in the test suite to run, either `dnf` or `createrepo_c`? I'd even argue it should have no default and be mandatory to specify which suite to run - they are in effect equal, even though `dnf` is what we want to run most of the time. (I'm not opposed to the default if others want to have it though)

I'd also move `dnf-behave-tests/features` to `dnf-behave-tests/dnf`, not necessarily as part of this PR, but I think it will make for a more clear naming. I don't think it should break anything?

Reply:
The --featuresdir is renamed to --suite. Also moving features/ to dnf/ makes sense, but I'd rather do this in another PR.